### ISSUE TITLE
${MAGMA}/apply_patches.sh failing on libxml2, php, poppler

### ISSUE THREAD

**Issue by anaaktge** [HexHive/magma#32]
I pulled from dev( lovely to see all the report work landing :) ) and ran into some issues building the targets. These all look similar. In that it looks like patching failed. openssl also failed, but that looks like a different issue and merits its own ticket. 

../log/afl_libxml2_build.log 
```
Applying /magma/targets/libxml2/patches/bugs/AAH036.patch
patching file HTMLparser.c
Hunk #1 FAILED at 2787.
Hunk #2 FAILED at 2804.
Hunk #3 FAILED at 2822.
Hunk #4 FAILED at 2847.
Hunk #5 FAILED at 2854.
Hunk #6 FAILED at 2867.
Hunk #7 FAILED at 2882.
7 out of 7 hunks FAILED -- saving rejects to file HTMLparser.c.rej
The command '/bin/sh -c ${TARGET}/fetch.sh && ${MAGMA}/apply_patches.sh' returned a non-zero code: 1
```

/log/afl_php_build.log
```
Applying /magma/targets/php/patches/bugs/MAE015.patch
patching file ext/exif/exif.c
Hunk #1 FAILED at 3941.
Hunk #2 FAILED at 3962.
Hunk #3 succeeded at 4745 (offset 42 lines).
2 out of 3 hunks FAILED -- saving rejects to file ext/exif/exif.c.rej
The command '/bin/sh -c ${TARGET}/fetch.sh && ${MAGMA}/apply_patches.sh' returned a non-zero code: 1
```

/log/afl_poppler_build.log
```
Step 44/59 : RUN ${TARGET}/fetch.sh && ${MAGMA}/apply_patches.sh
 ---> Running in fc4a5ab1dc5b
^[[91mCloning into '/magma/targets/poppler/repo'...
^[[0m^[[91mCloning into '/magma/targets/poppler/freetype2'...
^[[0m^[[91mfind: '/magma/targets/poppler/patches/setup': No such file or directory
^[[0mApplying /magma/targets/poppler/patches/bugs/AAH051.patch
patching file poppler/Parser.cc
Hunk #1 FAILED at 283.
1 out of 1 hunk FAILED -- saving rejects to file poppler/Parser.cc.rej
The command '/bin/sh -c ${TARGET}/fetch.sh && ${MAGMA}/apply_patches.sh' returned a non-zero code: 1
```

**Comment by hazimeh** [comment]
Target repository references are not anchored on the `dev` branch. They are only anchored on versioned branches, of which exists only one right now: `v1.0`.

Please use that branch for building the benchmark. The report generation tooling is still on the `dev` branch however, as it is incomplete. Once the tooling is finished, it will be merged into the latest versioned branch.

Let me know if you face any issues with `v1.0`
